src/tools/pub/package-layout.md

```diff
@@ -298,18 +299,18 @@ enchilada/
     style.css
 {% endprettify %}
 
-Dart is a web language, so many pub packages will be doing web stuff. That
-means HTML, CSS, images, and, heck, probably even some JavaScript. All of that
-goes into your package's `web` directory. You're free to organize the contents
-of that to your heart's content. Go crazy with subdirectories if that makes you
-happy.
+For web packages, place HTML, CSS, JavaScript, and Dart code under
+the `web` directory. Any Dart web entrypoints (in other words,
+Dart scripts that are referred to in a `<script>` tag) go under `web` and
+not `lib`. This ensures that `package:` imports can be resolved correctly.
+You can organize the `web` directory into subdirectories if you like.
 
-Also, and this is important, any Dart web entrypoints (in other words, Dart
-scripts that are referred to in a `<script>` tag) go under `web` and not `lib`.
-That ensures that `package:` imports can be resolved correctly.
+You can place assets (such as images) in any directory in the root package,
```

Comment:
I don't know what this means, but it sounds wrong. What's a "root package"? Why did you add assets to this section?

I think we used to think that everything that'd be compiled into your final app (not tests and tools, e.g.) could go under web/. We've changed to thinking that only entrypoints should go in web/, and everything else that's compiled into the final app should be under lib/. And more recently, that you should use lib/src for everything that won't be imported directly by something in web/ or by another package.